### PR TITLE
UI- node deployment progress

### PR DESCRIPTION
**Pull request by carlito-scality**
**Component**: UI

**Context**: 
Actually, there are no ways to track the progress of the deployment and no notification to give some feedback after an action is performed

**Summary**:
-Add "Node deployment" page to log the "new" and "completed" events of given job.
-Use localstorage to store jid/node_name to manage the refresh => it is a temporary solution. 

Limitations that will be dealt later:
-Multi-users management
-Use Custom event to fetch all deployment steps which will be displayed in a new page.

**Acceptance criteria**: 
Screenshot 2019-05-23 at 17 11 17
Screenshot 2019-05-21 at 16 46 36

### CODE REVIEW

File: ui/src/translations/fr.json [hunk 1]
```diff
@@ -33,5 +33,6 @@
   "ready": "Prêt",
   "not_ready": "Pas prêt",
   "deploy": "Déployer",
-  "version": "MetalK8s Version"
+  "version": "MetalK8s Version",
+  "node_deployment": "Déployement du node"
```
Comment (ramanduh) [inline ui/src/translations/fr.json]: typo: déplo**i**ement

File: ui/public/config.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-    "url": "https://172.21.254.13:6443", 
-    "url_salt": "http://172.21.254.13:4507"
+  "url": "https://172.21.254.14:6443",
+  "url_salt": "http://172.21.254.12:4507"
```
Comment (gdemonet) [inline ui/public/config.json]: Should this be committed?

File: ui/src/services/api.js [hunk 3]
```diff
@@ -143,3 +143,21 @@ export async function deployNode(url, token, node, version) {
     return { error };
   }
 }
+
+export async function lookupJid(url, token, jid) {
```
Comment (gdemonet) [inline ui/src/services/api.js]: `printJob` would be more accurate?

File: ui/src/ducks/app/nodes.js [hunk 4]
```diff
@@ -123,18 +205,53 @@ export function* deployNode({ payload }) {
...
+        message: result.error
+      })
+    );
   } else {
+    updateJobLocalStorage(result.data.return[0].jid, payload.name);
     yield call(fetchNodes);
   }
 }
 
+export function subSSE(eventSrc) {
```
Comment (gdemonet) [inline ui/src/ducks/app/nodes.js:222]: What does `SSE` stand for?

**Review by gdemonet — COMMENTED** [review]
Code looks good, just a thought: would be nice if we start to split explicitly between K8s API and SaltAPI (mentioning _jobs_ in `services.utils` for instance can be confusing, since the concept of _job_ exists in both APIs).

Will test it now, can you rebase your PR on `development/2.0`? #1144 landed, so control-plane expansion should be fixed.

File: ui/src/ducks/app/nodes.js (hunk 4 above)
Comment (carlito-scality) [reply ui/src/ducks/app/nodes.js:222]: Server sent event

File: ui/public/config.json (hunk 2 above)
Comment (carlito-scality) [reply ui/public/config.json]: nope :)

**Comment by carlito-scality** [comment]
@bert-e reset

**Comment by carlito-scality** [comment]
@bert-e reset

**Comment by carlito-scality** [comment]
@bert-e force_reset

**Review by gdemonet — APPROVED** [review]
Works as advertised, not really user-friendly (we loose the `new` event, and get a huge JSON in case of failure), but fine for now.

**Comment by carlito-scality** [comment]
@bert-e approve

**Pull request merged**